### DISCUSSION TITLE
Limit of number of claims ?

### DISCUSSION THREAD

**Discussion by Lavacobra** [GriefPrevention/GriefPrevention#2207] [Q&A]
Hi I hope you're doing good. So I wanted to add GriefPrevention on my server and I saw you can set a maximum of claimable blocks per player, but I was wonder if it is possible to do a number of claimed territories for each player that you can modify. For example :

Player can do 2 claims maximum but wants a third one, he buys an item and gives it to the staff so now the staff can give this specific player a third claim slot he can use.

I want to use it on a Role-Play server where players could claim their cities and if they want to claim other smaller cities they make, they can.

I am also wondering if this plugin has a discord server ? I couldn't find it... Have a nice day !

**Comment by QarthO** [comment]
Hi, what you want is not possible natively thru GP, but is definitely possible utilizing GPs api. So a 3rd party add-on can easily do what you want.

There is a discord. You can find it in the description on the main page of this github
https://r.griefprevention.com/dumcord/
